Plugin.php

```diff
@@ -64,6 +65,26 @@ public function register()
             $model->extendClassWith('RainLab\Translate\Behaviors\TranslatableModel');
         });
 
+        /*
+         * Add translation support to theme settings
+         */
+        ThemeData::extend(static function ($model) {
```

Comment:
Why the "static" keyword here?

Reply:
It is just a small optimisation. Static closures won't utilize the outer scope and therefore they are faster:

![image](https://user-images.githubusercontent.com/3897579/75520603-d4cca200-59fd-11ea-9838-247c6a0c5186.png)

Reply:
nice, didn't know that.